docs/getting-started.md

```diff
@@ -461,61 +500,92 @@ curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/stargaze
+            <version>1.3.1</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.6.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+
+            <!-- Build an executable JAR -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.0.2</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <classpathPrefix>lib/</classpathPrefix>
+                            <mainClass>main.java.StarTrace</mainClass>
+                        </manifest>
+                    </archive>
+                </configuration>
+            </plugin>
+
+            <!-- create an uber JAR -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+
+</project>
+```
+
+We will now create the java directory that will contain our `StarTrace.java` file and create the `StarTrace.java` file:
+```
+mkdir -p src/main/java && cd src/main/java
+touch StarTrace.java
 ```
 
 This file will be used in the following sections.
 
 ##### Create the Schema
 
-Before we can import data or run queries, we need to create our schema. The following imports can be seen from the java-pilosa repo:
-```
-import com.pilosa.client.PilosaClient;
-import com.pilosa.client.QueryResponse;
-import com.pilosa.client.exceptions.PilosaException;
-import com.pilosa.client.orm.*;
-import com.pilosa.client.csv.FileRecordIterator;
-import com.pilosa.client.TimeQuantum;
-```
-Create the schema by creating a client which will communicate our schema to Pilosa, creating a schema which will contain our indexes and fields, and syncing with Pilosa. This is all done in the `startrace.java` file:
+Before we can import data or run queries, we need to create our schema. The first 6 dependencies are imported from the java-pilosa library. Create the schema by creating a client which will communicate our schema to Pilosa, creating a schema which will contain our indexes and fields, and syncing with Pilosa. This is all done in the `StarTrace.java` file:
```

Comment:
"6" should be "six". This is one of those silly style rules that I don't really believe in, but follow compulsively.

This paragraph can also be changed to match the update in the corresponding part of the Go section (line 297)

Reply:
I changed it to be "You can see the first six dependencies are imported from the java-pilosa library." I also made the same change to the Python section